src/js/providers/html5.js

```diff
@@ -248,7 +250,8 @@ define([
 
         function _updateDuration(duration) {
             _duration = duration;
-            if (_delayedSeek && duration && duration !== Infinity) {
+            // Don't seek when _delayedSeek is set to -1 in _completeLoad
+            if (_delayedSeek && _delayedSeek !== -1 && duration && duration !== Infinity) {
                 _this.seek(_delayedSeek);
```

Comment:
@robwalch would this work with [HAVE_METADATA](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/readyState)? Description says: 'Seeking will no longer raise an exception.'

Reply:
Which description?

The issue is that this hack was added, without consideration of the code above:
https://github.com/jwplayer/jwplayer/blob/d30fee0e70ef211b4a0700787f65830fbe758892/src/js/providers/html5.js#L459

Since we're using `-1` to prevent seeking there, we should not allow it in `_updateDuration`.

Reply:
Regarding " HAVE_METADATA" readyState, we don't use it to determine when we can seek - that is not reliable. We use seekable ranges, duration and the "canplay" or "timeupdate" media element events (whichever comes first).